sampler/stories/default/IconItem.js

```diff
@@ -13,8 +13,8 @@ storiesOf('Agate', module)
 			<IconItem
 				label={text('label', IconItem, 'Label')}
 				disabled={boolean('disabled', IconItem)}
-				icon={select('icon', [...iconNames], IconItem, 'compass')}
-				titleIcon={select('titleIcon', ['', ...iconNames], IconItem, '')}
+				iconAfter={select('iconAfter', [...iconNames], IconItem, 'compass')}
```

Comment:
We should probably undo this and fix the shim. I didn't intend to change the props. :/